Release notes for 2.14.7 (#11262)
PrefectHQ/prefect: RELEASE-NOTES.md

# Prefect Release Notes

## Release 2.14.7

### Enhancements
- Use new route to read work pool types when connected to Prefect Cloud — https://github.com/PrefectHQ/prefect/pull/11236
- Add `parent_flow_run_id` to client schema — https://github.com/PrefectHQ/prefect/pull/11089

### Fixes
- [deploy ux] fix install when `requires` more than one package — https://github.com/PrefectHQ/prefect/pull/11254

### Documentation
- Add serverless work pool landing page — https://github.com/PrefectHQ/prefect/pull/11004
- Update Azure Container Instance guide to reflect current Azure Portal interface and Prefect UI — https://github.com/PrefectHQ/prefect/pull/11256

> - update imports — https://github.com/PrefectHQ/prefect/pull/11235

Reviewer: this is my bad for the ambiguous commit message
```suggestion
- Update `State` import to prefer `prefect.states` over server schemas — https://github.com/PrefectHQ/prefect/pull/11235
```

# Prefect Release Notes

## Release 2.14.7

### Enhancements
- Use new route to read work pool types when connected to Prefect Cloud — https://github.com/PrefectHQ/prefect/pull/11236
- Add `parent_flow_run_id` to client schema — https://github.com/PrefectHQ/prefect/pull/11089

### Fixes

> - [deploy ux] fix install when `requires` more than one package — https://github.com/PrefectHQ/prefect/pull/11254

Reviewer 2: probably want to make this one more user-facing. `prefect deploy` I'm guessing

### Documentation
- Add serverless work pool landing page — https://github.com/PrefectHQ/prefect/pull/11004
- Update Azure Container Instance guide to reflect current Azure Portal interface and Prefect UI — https://github.com/PrefectHQ/prefect/pull/11256

> - update imports — https://github.com/PrefectHQ/prefect/pull/11235

Reviewer 2: imports of what?

> ## New Contributors
> * @oakbramble made their first contribution in https://github.com/PrefectHQ/prefect/pull/11089
> 
> ### Contributors
> - @oakbramble
> 
> 

Reviewer 2: ```suggestion
## New Contributors
* @oakbramble made their first contribution in https://github.com/PrefectHQ/prefect/pull/11089

```

# Prefect Release Notes

## Release 2.14.7

> This release fixes a bug introduced in 2.14.6 where deployments with default Docker image builds looked for images tagged `v2.14.6` instead of `2.14.6`.

Author: Does this fix builds for 2.14.6 or for builds that look for latest?

Reviewer 3: Users with 2.14.6 will still experience the issue after this release.

After this release, if a user has 2.14.7 installed, the deployment creation Docker build step should look for 2.14.7.

Deployment creation with 2.14.5 works fine now (specifies an image tagged 2.14.5).

> This release fixes a bug introduced in 2.14.6 where deployments with default Docker image builds looked for images tagged `v2.14.6` instead of `2.14.6`.

Reviewer 3: ```suggestion
This release fixes a bug introduced in 2.14.6 where deployments with default Docker image builds looked for images tagged `v2.14.6` instead of `2.14.6`. Users of `2.14.6` should upgrade if planning to create deployments with an image other than a custom image.
```

### Enhancements

> - Use new route to read work pool types when connected to Prefect Cloud — <https://github.com/PrefectHQ/prefect/pull/11236>

Reviewer 4: ```suggestion
- Use a new route to read work pool types when connected to Prefect Cloud — <https://github.com/PrefectHQ/prefect/pull/11236>
```

> - Add `parent_flow_run_id` to client schema — <https://github.com/PrefectHQ/prefect/pull/11089>

Reviewer 4: ```suggestion
- Add `parent_flow_run_id` as a new API filter for flow runs — <https://github.com/PrefectHQ/prefect/pull/11089>
```
